pkg/webhooks/errors.go

```diff
@@ -37,9 +39,44 @@ func IsAdmissionCantConnect(err error) bool {
 	return false
 }
 
+func IsCantConnectAPIServer(err error) bool {
+	if apierrors.IsInternalError(err) && strings.Contains(err.Error(), cantConnectAPIServerMsg) {
+		return true
+	}
+	return false
```

Comment:
Would this be more straightforward?  (same for the other ones that follow this pattern)
```suggestion
	return apierrors.IsInternalError(err) && strings.Contains(err.Error(), cantConnectAPIServerMsg)
```

Reply:
I don't know which one, if any, is more idiomatic / Go-like.

I'm down to update those function definitions if it's deemed better / easier to read and understand.

Reply:
Brevity is probably good in this case. Thanks!

Done in a99daa5a12a2e2d172c964f4e7f4b360a6ae1816.